### ISSUE TITLE
[amenity-points] New icons

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#3156]
**[Submitted to the original trac issue database at 9.55pm, Wednesday, 4th August 2010]**

The last project of the week showed, that people are interested in more icons.
http://wiki.openstreetmap.org/wiki/Project_of_the_week/2010/Jul_25
Even if I understand the need of a soft control to get no ugly layout I would like to see more icons for several reasons:

1. logical misses
If we display BAKERY the enduser will not be aware, that we decided to hide BUTCHER at the moment. Candidates are:
-SHOP=MALL
-SHOP=BUTCHER
-SHOP=SHOES

2. clustering
Some icons appear a lot of times and make it difficult to the enduser to take a choice. Example: PARKING_PLACE with subtypes for multistory, underground, park&ride which help a lot to decide on higher zoom levels where to park your car when you are a stranger.

3. improved grouping
Even if icons consume space, they allow to save space and clear up the design by using smaller fonts (e.g. SHOP=*) and use the same font colours.

4. isolated icons
Some features are far away from others places, only. e.g. CREMATORY at a cementry
candidates:

5. show what's in the DB
For endusers new to OSM the default mapnik map 'is' OSM. They are unable to see behind and understand the idea of a DB with multiple map views (and it's problems) at the very first beginning. So we have to take the chance to show them at this moment what OSM allready has. This is on the one hand completeness (so a mass of objects) and on the other hand a high level on detail (a mass of different object groups). This includes candidates that impress novices and turn to a 'how clever!' effect e.g. SHOP=LAUNDRY

6. very important icons
Last but not least there are icons everybody new just expected and that are useful 'for the most people'
e.g. SHOP=CAR, SHOP=CAR_REPAIR, SHOP_BICYCLE

I know it's a real hard choice, we might ask the whole community for what is important to them.

**Comment by tomhughes** [comment]
**Author: Ldp**
**[Added to the original trac issue at 11.05pm, Friday, 24th September 2010]**

1. shop=butcher was added 5 weeks ago (r22730, r22738)

2. There were some objections to diversification of parking symbols. Perhaps that has changed. I'll ask around.

3. shop texts are already slightly smaller than other POI texts.

4. The POTW hasn't produced an icon for this yet.

5. The default map can and will never show every type of POI we can come up with as mappers. I mentioned that before, on the forum.

6. These three were added today (r23338, r23339).

Importance is relative, but the database is very helpful as well. When we started adding shops, we looked at tagstat/tagwatch to decide which were the most proliferous, and then took the first 5 of that list, and later added another 5.
